README

```diff
@@ -1,7 +1,95 @@
-Download the graphics, font, and sound files to be used in your code
-https://drive.google.com/drive/folders/1WXHI-pd8VQT9KvmAx280LxZfvcfG7jOh?usp=sharing
+"""
+Project: Python + Game
+Lead: Bernard Yap
+Team Members: Benjamin Goh, Nazrin Nazarudin, Yun Sion, Luke Leh, Lim Eejoy, Guan Zhou, Tarun, Raja Darshini
+Goal: Clone the original mario game as much as possible!
 
-Happy coding! Bernard will be your supervisor, just ask me if anything 
+Credits to justinmeister for all the resources
+https://github.com/justinmeister/Mario-Level-1/tree/master/resources/graphics
+
+click on the link below to see the real mario game
+https://supermarioplay.com/
+"""
+import pygame
+import sys
+
+#initializing pygame
+pygame.init()
+
+# create a screen instance/object with 
+# 1000px width and 448px height
+screenW = 1000
+screenH = 448
+screen = pygame.display.set_mode((screenW,screenH))
+
+
+# I forgot to mention 'Clock'
+# the 'clock' instance help us to control the FPS
+# see below - clock.tick(60) makes the fps to be 60 
+clock = pygame.time.Clock()
+
+# import images & animation here
+bg = pygame.image.load("C:\\Users\\nazri\\Downloads\\Mario\\static_images\\background.png")
+mario = pygame.image.load("C:\\Users\\nazri\\Downloads\\Mario\\static_images\\mario.png")
+
+# Tips: The bricks (floor) is 48px height
+
+# import sounds here
+
+# import fonts here
+
+#variables
+marioW = 24    #mario width
+marioH = 32    #mario height
+bg_x_pos = 0   #background position
+bgW = 6784     #bacgkround width
+bgH = 448      #background height
+bgLimit = screenW - bgW
+mario_x_pos = 0
+mario_y_pos = 368
+marioWalkR =  False
+marioWalkL = False
+marioSpeed = 10
+
+
+
+#event
+while True:
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+            sys.exit()
+
+        if event.type == pygame.KEYDOWN:     #user pressed a key      
+            if event.key == pygame.K_d:      #user pressed the D key
+                marioWalkR = True
+            if event.key == pygame.K_a:      #user pressed the A key
+                marioWalkL = True
+        
+        if event.type == pygame.KEYUP:       #user released a key
+            if event.key == pygame.K_d:      #user released the D key
+                marioWalkR = False
+            if event.key == pygame.K_a:      #user released the A key
+                marioWalkL = False
+
+    #Mario walk right logic       
+    if marioWalkR == True:
+        if bg_x_pos > bgLimit:                   #logic before limit is reached
```

Comment:
I don't think you need to care about the end screen limit, since mario won't reach the end screen anyway
If you play the mario game at the end mario will jump off the pole and walks into the castle :D 
We will code that soon